appveyor.yml

```diff
@@ -8,27 +7,51 @@ platform:
   - x86_64
 
 cache:
-  - "C:\\SR"
-  - dist-newstyle
+  - "%STACK_ROOT% -> %STACK_YAML%, appveyor.yml"
+  - ".stack-work -> %STACK_YAML%, appveyor.yml"
 
 environment:
-  global:
-    CABOPTS: --store-dir=C:\\SR
+  STACK_ROOT: C:\sr
+  STACK_VERSION: 2.1.1
+
+  # Workaround a gnarly bug https://github.com/haskell/cabal/issues/5386
+  # See: https://www.fpcomplete.com/blog/2018/06/sed-a-debugging-story
+  # TODO: check if it's fixed once we switch to lst-13 and GHC 8.6
```

Comment:
That might be an outdated comment :slightly_smiling_face:

Reply:
@vrom911 Good point! I'll remove it 👍